patches/minecraft/net/minecraft/network/login/server/SCustomPayloadLoginPacket.java.patch

```diff
@@ -3,9 +3,44 @@
 @@ -8,7 +8,7 @@
  import net.minecraftforge.api.distmarker.Dist;
  import net.minecraftforge.api.distmarker.OnlyIn;
- 
+
```

Comment:
My IDE must of had a heart attack at seeing FF's code style. I've seen it and will fix when I have a chance.